### PR TITLE
Caught up with latest baresip updates

### PR DESCRIPTION
**Pull request by Koshub**
- changed `rem` from `https://github.com/creytiv/rem.git` to `git clone https://github.com/baresip/rem.git`
- removed `rem` patch
- added OPENSSL_OPT override. See #42 
- added changes to use last configurations around libs versions management

### CODE REVIEW

**Comment by Koshub** [comment]
@sreimers @alfredh

File: re-openssl-opt.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+diff --git a/mk/re.mk b/mk/re.mk
+index 788943f..d626a29 100644
+--- a/mk/re.mk
++++ b/mk/re.mk
+@@ -427,7 +427,8 @@ endif
+ # External libraries section
+ #
+ 
+-OPENSSL_OPT := $(shell [ -f /usr/local/opt/openssl/include/openssl/ssl.h ] \
++OPENSSL_OPT :=
```
Comment (sreimers) [inline re-openssl-opt.patch:10]: This should not be needed, you can call `make OPENSSL_OPT=` to override

Comment (Koshub) [reply re-openssl-opt.patch]: @sreimers thanks. Done

**Comment by sreimers** [comment]
Thanks

**Pull request merged**